packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -52,6 +54,43 @@ import {
 const ACTIVITY_CACHE_ID = 'activity';
 export const LAST_PAGE_REACHED = 'last_page_reached';
 
+/**
+ * Calculate fiat value for a token amount
+ * @param symbol - Token symbol
+ * @param value - Token amount as string
+ * @param decimals - Token decimals
+ * @param tokenPrices - Array of token prices
+ * @param currency - Target fiat currency
+ * @returns Fiat value object or undefined if price not found
+ */
+const calculateFiatValue = (
+  symbol: string,
```

Comment:
Ideally, we should be using the contractAddress here instead of symbols. Multiple deployed token contracts can have same the symbol.

Reply:
For fiat values its tricky to give only token address as we don't get a fiat prices on testnets and if we use those contract addresses it won't work

Reply:
I'll implement it in the same way we currently do for Balance tab inside UI.

Reply:
yeah we can make a map `testtokenaddr->mainnettokenaddr`
```ts
const TOKEN_PRICE_MAP = {
  // testnet address: mainnet address
  "0xTestnetTokenAddr1": "0xMainnetTokenAddr1",
  "0xTestnetTokenAddr2": "0xMainnetTokenAddr2",
};
```